### PR TITLE
Profile Creation 

### PR DESCRIPTION
**Pull request by diksha1159**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7622"></a>

### CODE REVIEW

File: diksha1159.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Diksha Jadhav",
+  "bio": "Just a CS student trying to learn new exciting stuff"
```
Comment (RohanSasne) [inline diksha1159.json:3]:
```suggestion
  "bio": "Just a CS student trying to learn new exciting stuff",
```

Include a `,` to parse the json correctly

**Review by RohanSasne — CHANGES_REQUESTED** [review]

**Review by SaraJaoude — APPROVED** [review]
Please move your file to the `data` folder. 

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request.

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please move your file to the `data` folder. 

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request.

**Comment by diksha1159** [comment]
hi please help me understand the problem here
I think the format here seems correct 

take a look please 
image

**Comment by eddiejaoude** [comment]
> how to move the file to the data folder? or should I create a new file instead? or will a simple drag and drop help?

You can edit the file (this includes the path), and you can learn more about it from our docs (there is a there video at the top of the docs file also)

Any questions let us know

**Pull request merged**